packages/firestore/test/unit/remote/node/serializer.test.ts

```diff
@@ -421,60 +324,42 @@ describe('Serializer', () => {
       const serializer = new JsonProtoSerializer(partition, {
         useProto3Json: true
       });
-
-      const value: fieldValue.FieldValue = new fieldValue.BlobValue(example);
-      const proto: api.Value = expected;
-      const protobufjsField = new Uint8Array(bytes);
-
-      expectFieldValueJsonProtoRoundTrip(value, proto, serializer);
-      expectJsonProtoProtobufjsMessageRoundTrip(
-        proto,
-        protobufjsField,
-        'bytesValue'
-      );
-      expectFullRoundTrip(value, serializer);
+      const obj = serializer.toValue(new fieldValue.BlobValue(example));
+      expect(obj).to.deep.equal(expected);
```

Comment:
Undoing the call to verifyFieldValueRoundTrip essentially means we never test the round trip of a fieldValue.BlobValue with proto3. Is this ok? (I'm not sure how/where we use proto3json, so I wasn't sure.)

Reply:
Oh, that's a fair point.  I thought I was reverting this to the state in master but I forgot there's a "converts BlobValue from string" test in master that was removed in your PR...  That said, I think your PR wasn't testing the full round-trip either, since it included round-tripping through protobufJS which yields a UInt8Array instead of base64 string (and this is why you had to remove that assert in serializer.ts).

So I tweaked verifyFieldValueRoundTrip() to accept a `useProto3Json` option that uses the proto3 JSON serializer and skips the round-trip through protobufJS.